### ISSUE TITLE
doesn't seem to work on SteamDeck

### ISSUE THREAD

**Issue by aebrer** [SchuhBaum/SBCameraScroll#6]
I followed the various troubleshooting steps people have shared, but nothing I do seems to be able to get this working on the SteamDeck. Could it be something to do with the resolution of the display?

I was able to confirm it works on my windows PC (and is an amazing mod haha)

**Comment by SchuhBaum** [comment]
Hey Andrew!

SteamDeck uses Linux and there are additional steps there before you can
use mods. I am not too familiar with it but I saw extensive threads for the
SteamDeck on the RainWorld Discord server (https://discord.gg/rainworld).
Maybe you can check these out.

Best regards,
SchuhBaum.

**Comment by aebrer** [comment]
Dug in deeper than last time based on your comment and found this: 

"So, I'm a Steam Deck user, and mods are working fine for me. I suspect this is because of the launch options I set in Steam properties for Rain World for a hibernation bug fix a while ago.

For some reason, on Linux BepInEx mods didn't actually get loaded until I added `WINEDLLOVERRIDES="winhttp=n,b" %command%` to the launch options. Probably an issue with Proton.

Maybe try that?"

https://discord.com/channels/291184728944410624/1073887200841834556/1076196962807861359

Tested and confirmed this works! Just had to set this launch option, then disablr and re-enable the mod. Thanks for the help!
